ansible/roles/networking/tasks/main.yaml

```diff
@@ -1,124 +1,157 @@
-  - name: load br_netfilter kernel module
-    modprobe:
-      name: br_netfilter
-      state: present
-    register: br_netfilter
+- name: load br_netfilter kernel module
```

Comment:
Fixed indentation on this file. somehow it was everything was indented at 4 spaces.